pal/config.py

```diff
@@ -155,10 +156,6 @@ def validate(self):
 # Function generation options
 # -----------------------------------------------------------------------------
 
-#  c = Configuration("encoded_functions", False)
-#  c.description = "Use 32bit encoded instruction values instead of traditional mnemonic accesses"
-#  config.add_configuration(c)
-
```

Comment:
Is the plan to remove binary encoded functions or just hold off for now? I know that it is a lot more complex in the Intel environment, so it is kind of unique to the ARM architecture itself. I think the encoded functions are useful, but I am not sure how they fit in to the new PAL.

Reply:
The plan isn't to remove support for the binary encoded functions, the plan is to refactor that support into a BinaryEncodedAccessMechanismWriter class. That would be another good issue to open, I'll do that now.